lib/order.rb

```diff
@@ -0,0 +1,32 @@
+class Order
+
+  attr_reader :orderlines, :total
+
+  def initialize
+    @orderlines = nil
+    @total = 0
+  end
+
+  def create_order(order_hash)
+    @orderlines = order_hash
+    calculate_total
+  end
+
+ private
```

Comment:
Ah - you reminded me I need to use private and public methods :) Thanks for the reminder!